### ISSUE TITLE
Questions about Finetuning part

### ISSUE THREAD

**Issue by SII-Ferenas** [bcmi/CaGNet-Zero-Shot-Semantic-Segmentation#17]
Hi, thanks for this excellent work! However, I have one problem with the settings in Finetuning part. In this paper, you mention "...The corresponding label map is Y(s/m) with pixel-wise label vector...", which contains both the unseen and seen part. Does it mean the ground-truth label map of the unseen class is involved during the finetuning process? Otherwise how to generate the label map of the unseen class during the fine-tuning process?
